static/js/LexiconBox.jsx

```diff
@@ -145,8 +146,8 @@ class LexiconBox extends Component {
               ) : null
             }
             <div className="contentText named-entity-description">
-              <span className="en">{ne.description ? ne.description.en : `No description known for '${ne.primaryTitle.en}'`}</span>
-              <span className="he">{ne.description ? ne.description.he : `לא קיים מידע עבור '${ne.primaryTitle.he}'`}</span>
+              <InterfaceText markdown={{en: ne.description ? ne.description.en : `No description known for '${ne.primaryTitle.en}'`,
```

Comment:
looks great, no playwright test coverage as of now, please write test to verify LexiconBox.jsx open properly and can evaluate markdown properly